print-battleground-state-changes

```diff
@@ -543,11 +563,12 @@ def rss_output(path, summarized):
         ), file=rssfile)
 
 rss_output('battleground-state-changes.xml', battlegrounds_summarized)
+rss_output('all-state-changes.xml', summarized)
 
 # this file is deprecated and should not be used! it will be removed soontm
```

Comment:
Is there any reason that we can't remove `notification-updates.json`?

Reply:
Nope. Happy to do so.

Reply:
I think we do that in another PR though. We can keep reporting the battleground state results in that file and then delete it separately in case it causes problems?